Mergin/plugin.py

```diff
@@ -205,6 +205,16 @@ def _unsaved_changes_check(self):
             return True
         return True
 
+    def _check_writing_permissions(self):
+        """Check if user have writing rights to the project."""
+        info = self.mc.project_info(self.project_name)
+        username = self.mc.username()
+        writersnames = info["access"]["writersnames"]
+        if username not in writersnames:
+            return False
+        else:
+            return True
```

Comment:
`return username in writersnames` would be shorter :-)

Reply:
Done.